app/jobs/importing/run_daily_imports_job.rb

```diff
@@ -251,5 +253,31 @@ def sync_with_cas
       GrdaWarehouse::Tasks::PushClientsToCas.new.sync!
       @notifier.ping('Pushed Clients to CAS')
     end
+
+    # UNTESTED partial implementation. This should probably go in a utility class somewhere else
+    def collect_postgres_stats
+      Dir.mktmpdir do |dir|
+        GrdaWarehouse::PostgresInspector.inspect_each do |inspector|
+          [
+            :database_stats,
+            :table_stats,
+            :index_stats,
+          ].each do |stats_method|
+            results = inspector.send(stats_method)
+            filename = Pathname.new(dir).join("#{inspector.name}.#{stats_method}.csv")
+            CSV.open(filename.to_s, 'w') do |csv|
+              headers = results.first.keys
+              csv << headers
+
+              results.each do |result|
+                csv << result.values_at(**headers)
+              end
+            end
+          end
+        end
+        raise 'implementation tbd'
+        # send_directory_to_s3(dir)
```

Comment:
These should be dropped into a file on S3 in the metrics bucket (issue opened with devops) with a path something like:
```
CLIENT-ENV-APP/YYYY-MM-DD--CONNECTION-METHOD.csv
```

Example: `qa-staging-warehouse/2024-01-12--staging_qa_warehouse-database_stats.csv`

We'll probably want to setup an RemoteCredential that knows how to find the location.

Reply:
* I put the s3 export logic into AppResourceMonitor::Report
* job will sync to s3 if there's a remote credential with slug for app_stats
* added timestamp, client, and RAILS_ENV to path/file: `/qa_hmis-staging/postgres_database_stats-20240113180926.csv